portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,252 @@
+                and trained younger matheletes in middle and high school. I also was a top contributer to Mathematical Stack Exchange. <br>
+                I like to read, take photos, and travel. <br> <br>
+
+                If it sounds like we might get along, let me know! :D
+
+            </p>
+        </div>
+        </div>
+    </section>
+
+    <!--Photography-->
+    <!--want to make this an assorted randomized photo tile -->
+    <section>
+        <h1>Photography</h1>
+        <div class="slider">
+            <div class="slides">
+                <input type="radio" name="radio-btn" id="radio1">
+                <input type="radio" name="radio-btn" id="radio2">
+                <input type="radio" name="radio-btn" id="radio3">
+                <input type="radio" name="radio-btn" id="radio4">
+
+                <div class="photo first">
+                    <img src="images/photo1.jpeg" alt="">
+                </div>
+                <div class="photo">
+                    <img src="images/photo3.jpeg" alt="">
+                </div>
+                <div class="photo">
+                    <img src="images/photo2.jpeg" alt="">
+                </div>
+                <div class="photo">
+                    <img src="images/photo4.jpeg" alt="">
+                </div>
+
+                <div class="navigation-auto">
+                    <div class="auto-btn1"></div>
+                    <div class="auto-btn2"></div>
+                    <div class="auto-btn3"></div>
+                    <div class="auto-btn4"></div>
+                </div>
+            </div>
+            <div class="navigation-manual">
+                <label for="radio1" class="manual-btn"></label>
+                <label for="radio2" class="manual-btn"></label>
+                <label for="radio3" class="manual-btn"></label>
+                <label for="radio4" class="manual-btn"></label>
+            </div>
+        </div>
+    </section>
+
+    <!--Reads-->
+    <section>
+        <h1>Reads</h1>
+        <div>
+            <h3>A Recent Book I Read That I Recommend:</h3>
+            <ul>
+                <li>Cursed Bunny, by Bora Chung</li>
+            </ul>
+        
+            <h3>My All-Time Favourites</h3>
+            <ul>
+                <li>The Red Book, by Carl Jung</li>
+                <li>Cosmos, by Carl Sagan</li>
+                <li>The Little Prince, by Antoine de Saint-Exupéry</li>
+                <li>And Then There Were None, by Agatha Christie </li>
+                <li>Murder on the Orient Express, by Agatha Christie</li>
+                <li>The Socrates Express, by Eric Weiner</li>
+                <li>The Book Thief, by Markus Zusak</li>
+            </ul>
+
+            <form method="POST" action="/form-handler" onclick="bookrec()">
```

Comment:
Can't tell from the top of my head whats wrong on this form or if its just that the live website isn't up to date, but currently the code is being called as soon as I click on the text area

Reply:
Hi Estefany! Thank you so much for your comment! 

I wanted to make it so that when the user clicks on the submit button it shows "thank you" and below that also show "You recommended: book title + author", without having to reload the page or go to a new url etc. 

The submit/thank you part was a simple onclick function, but I'm having trouble with the second part. 

I tried to work on this (for reference: commit d3d2d1b) by using the POST fetch method (similar to what I've done in the About blurb translation), but mine still loads to a new url /form-handler, and I'm not sure how to resolve the issue.

I also tried async and fetch like I did with pokemons, but that's frankly looking a little worse (it's messy, and probably far from being correct, so I didn't commit). 

Do you know what I might be doing wrong?

Reply:
So I think it's because of the action part of the form (action="/form-handler") this will send you to that page onclick no matter what the javascript method does. so you can just remove that and make sure to fetch the response through your javascript method

Reply:
Thank you so much for the tip!

I believe I now have it fully working—please see commit [9241e4b] :D

(I'm frankly a little less than 100% sure since the POST logs in logs explorer don't show the actual contents of the submitted texts, but since I've used POST methods to retrieve the text content and display it on webpage, I believe it's working.)